sherpa/astro/io/crates_backend.py

```diff
@@ -952,8 +952,7 @@ def get_pha_data(arg, make_copy=True, use_background=False):
         # Chandra Level 3 PHA files
         for ii in range(phadataset.get_ncrates()):
             block = phadataset.get_crate(ii + 1)
-            hduclas2 = block.get_key('HDUCLAS2')
-            if hduclas2 is not None and hduclas2.value == 'BKG':
+            if (_try_hdr_key(block, 'HDUCLAS2') == 'BKG'):
```

Comment:
I hadn't realised this was found external to the sherpa tests, so +1 for having a test for this (especially as we have possible changes in the crates backend about the "string" type, so we want to make sure there's a test).